readme.md

```diff
@@ -105,6 +106,20 @@ ___
 ]) !!}
 ```
 
+##### Input File
+
+```php
+{!! Form::control('file', 'thumbnail', $errors, [
+    'label' => 'Your thumbnail',
+]) !!}
+```
+
+Or you can even use the input but And don't forget the 'files' => true in the Form::model() or Form::open():
```

Comment:
I think that "And don't forget the 'files' => true in the Form::model() or Form::open():" should be below this :

```php
{!! Form::file('thumbnail')  !!}
```